src/SFA.DAS.Payments.RequiredPayments.Tests.Specs/Extensions.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.Payments.RequiredPayments.Tests.Specs
+{
+    public static class Extensions
```

Comment:
This looks like a duplication of the helper methods in the End to End Acceptance Tests. I think we should create a tech debt ticket to remove this, once the Acceptance Tests are compatible with ,net core